### PR TITLE
Create PRS14317Rowe

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a record for W.G.H. Lowe (!), who sold the manuscript BLorient11601 to the BM (=BL).

### CODE REVIEW

File: new/PRS14317Rowe.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            <change who="CH" when="2024-01-31">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person>
+                    <persName><forename>W. G. H. </forename><surname>Lowe</surname></persName>
+                    <floruit notBefore="1870" notAfter="1970"/>
+                </person>
```
Comment (eu-genia) [inline new/PRS14317Rowe.xml:49]: he may have been a jeweller from Canada https://archive.org/stream/jewelerscircular00unse_64/jewelerscircular00unse_64_djvu.txt, one of the directors of the Commercial Travellers Association of Montreal https://numerique.banq.qc.ca/patrimoine/details/52327/4235959?docsearchtext=w.g.h.%20lowe (elected 1884) and member of the local masonic lodge; he died on 3 February 1904  https://dokumen.tips/document/proceedings-grand-lodge-of-af-am-of-canada-1904.html?page=336

**Comment by CarstenHoffmannMarburg** [comment]
I have no problem with creating zotero tags for the secondary literature investigated by Eugenia. Quoting the links may be not ideal, because it frequently causes problems with expired links.

File: new/PRS14317Rowe.xml [hunk 2]
```diff
@@ -46,7 +46,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <person>
                     <persName><forename>W. G. H. </forename><surname>Lowe</surname></persName>
                     <occupation type="other">He may have been a jeweller from Canada, one of the directors of the Commercial Travellers 
-                        Association of Montreal (elected 1884) and member of the local masonic lodge.</occupation>
+                        Association of Montreal (elected 1884) and member of the local masonic lodge according to
+                        <bibl><ptr target="bm:JewelersCircular1886"/><citedRange>63</citedRange></bibl>,
+                        <bibl><ptr target="bm:bm:TheTrader1885LoweAnderson"/></bibl>,
```
Comment (thea-m) [inline new/PRS14317Rowe.xml:51]: bm:bm

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

**Review by thea-m — APPROVED** [review]

**Pull request merged**